website/docs/docs/collaborate/column-level-lineage.md

```diff
@@ -54,3 +58,9 @@ Possible error cases are:
 - **Parsing error** &mdash; Error occurs when the SQL is ambiguous or too complex for parsing. An example of ambiguous parsing scenarios are _complex_ lateral joins.
 - **Python error** &mdash; Error occurs when a Python model is used within the lineage. Due to the nature of Python models, it's not possible to parse and determine the lineage.
 - **Unknown error** &mdash; Error occurs when the lineage can't be determined for an unknown reason. An example of this would be if a dbt best practice is not being followed, like using hardcoded table names instead of `ref` statements.
+
+### Data platform support
+
+CLL works with the dbt Cloud supported warehouses, including but not limited to Snowflake, BigQuery, Redshift, and Databricks (Unity Catalog). 
+
+CLL doesn’t currently support the Hive metastore version of Databricks, Apache Spark, Starburst/Trino, or Microsoft Fabric. Coming soon. 
```

Comment:
CLL in dbt Cloud works with the following data platforms: Snowflake, BigQuery, Redshift, and Databricks (Unity Catalog). 

The following adapters aren't yet supported by CLL in dbt Cloud: Hive metastore version of Databricks, Apache Spark, Starburst/Trino, and Microsoft Fabric. More of these platforms will be supported in the future.